### ISSUE TITLE
"WARNING: Failed to parse subvolume detail" when using quotas

### ISSUE THREAD

**Issue by MeisterP** [digint/btrbk#290]
I get the following warning:
```
WARNING: Failed to parse subvolume detail "Quota group: 0/5" for: /mnt/btrfs/rootfs
WARNING: Failed to parse subvolume detail "Limit referenced: -" for: /mnt/btrfs/rootfs
WARNING: Failed to parse subvolume detail "Limit exclusive: -" for: /mnt/btrfs/rootfs
WARNING: Failed to parse subvolume detail "Usage referenced: 16.00KiB" for: /mnt/btrfs/rootfs
WARNING: Failed to parse subvolume detail "Usage exclusive: 16.00KiB" for: /mnt/btrfs/rootfs
```

```
$ btrbk --version
btrbk command line client, version 0.28.3

$ btrfs --version
btrfs-progs v5.2.1 
```

I'm using the option `target_qgroup_destroy yes`

**Comment by digint** [comment]
Thanks for reporting!

This warning is "informative only", it tells me that `btrfs subvolume show` has some (new?) output which it does not know of, see: https://github.com/digint/btrbk/blob/v0.28.3/btrbk#L976-L1003

You can safely ignore these warnings, I will push a fix soon

**Comment by digint** [comment]
fixed in: 74be074e6f0347dd729ea450fd5811dc1500f2f8
